index.js

```diff
@@ -1,153 +1,159 @@
-"use strict"
+'use strict';
 
 module.exports = {
-    "env": {
-        "browser": true,
-        "node": false,
-        "es6": true
+    env: {
+        browser: true,
+        es2022: true,
     },
-    "globals": {
-        "require": true,
-        "module": true
+    extends: 'eslint:recommended',
+    plugins: ['import'],
+    parserOptions: {
+        sourceType: 'module'
     },
-    "extends": "eslint:recommended",
-    "parserOptions": {
-        "ecmaVersion": 6,
-        "sourceType": "module"
-    },
-    "rules": {
-        "indent": [
-            "error",
+    rules: {
+        'arrow-parens': ['error', 'always'],
+        'indent': [
+            'error',
             4,
             {
-                "SwitchCase": 1
+                SwitchCase: 1,
             }
         ],
-        "linebreak-style": [
-            "error",
-            "unix"
-        ],
-        "quotes": [
-            "error",
-            "single"
-        ],
-        "one-var": [
-            "error",
-            "never"
-        ],
-        "operator-linebreak": [
-            "error",
-            "before"
-        ],
-        "padded-blocks": [
-            "error",
-            "never"
-        ],
-        "semi": [
-            "error",
-            "always"
-        ],
-        "max-len": [
-            "error",
+        'quotes': ['error', 'single'],
+        'one-var': ['error', 'never'],
+        'operator-linebreak': ['error', 'before'],
+        'padded-blocks': ['error', 'never'],
+        'semi': ['error', 'always'],
+        'max-len': [
+            'error',
             120,
             {
-                "ignorePattern": "^import\\s.+\\sfrom\\s.+;$",
+                ignorePattern: '^import\\s.+\\sfrom\\s.+;$',
```

Comment:
would also add `ignoreComments` and `ignoreTrailingComments`

Reply:
@alexander-schranz Done, but `ignoreTrailingComments` is not necessary, as `ignoreComments` handles this already.

From the documentation of `ignoreComments`:
> true ignores all trailing comments and comments on their own line